frontend/public/components/_search.scss

```diff
@@ -1,5 +1,12 @@
 .co-search__accordion {
   padding-top: 0 !important;
+  // Align with search results
+  @media (min-width: $screen-sm-min) {
+    .co-search-group__accordion-toggle {
+      --pf-c-accordion__toggle--PaddingLeft: 30px;
+      --pf-c-accordion__toggle--PaddingRight: 30px;
```

Comment:
```suggestion
      --pf-c-accordion__toggle--PaddingRight: $grid-gutter-width;
```

Reply:
Seems that using `--pf-c-accordion__toggle--PaddingRight: $grid-gutter-width;` doesn't render, so I'm switching `padding-left` `padding-right` with the `$grid-gutter-width`